docs/en/05_Contributing/16_Maintainer_Guidelines.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Maintainer guidelines
+summary: Cheat sheet for module maintainers
+iconBrand: wpforms
+---
+
+
+# Maintainer Guidelines
+
+This doc is a cheat sheet for maintainers. It explains what that means to be a module maintainer and contains links to other resources with extra information.
+
+Module maintainers are people taking care of the repositories, CI, documentation, source code, conventions, community communications, release management etc.
+
+One of the most important maintainer responsibilities is to collaborate with other maintainers. Another important role is to facilitate community contributions (such as issue reports and pull requests).
+
+[note]
+A lot of extra information is available in Silverstripe CMS documentation section [“Contributing to Silverstripe”](./).
+All maintainers should be familiar with those docs as they explain many details about how we work.
+[/note]
+
+
+## What is Silverstripe Core
+
+The "core" of Silverstripe is a set of non-dev dependencies contained in
+[silverstripe/recipe-cms](https://github.com/silverstripe/recipe-cms)
+and [silverstripe/recipe-core](https://github.com/silverstripe/recipe-cms).
+It is limited to dependencies owned by the "silverstripe" Packagist vendor,
+but can include "dev" dependencies.
```

Comment:
```suggestion
Silverstripe CMS is powered by a system of components in the form of Composer packages. These packages will be categorised as either a _module_ or a _recipe_.

The "core" of Silverstripe refers to the module packages owned by the "silverstripe" Packagist vendor that fall under one of the following recipes:

* [silverstripe/recipe-core](https://github.com/silverstripe/recipe-cms)
* [silverstripe/recipe-cms](https://github.com/silverstripe/recipe-cms)
* [silverstripe/installer](https://github.com/silverstripe/silverstripe-installer)
```

I've pulled this information from https://docs.silverstripe.org/en/4/getting_started/recipes/ and thought it might be a bit more clear.